frontends/election-manager/src/app.test.tsx

```diff
@@ -66,7 +66,10 @@ jest.mock('./components/hand_marked_paper_ballot');
 beforeEach(() => {
   // we don't care about network errors logged to the console, they're crowding things
   jest.spyOn(console, 'error').mockImplementation(() => {}); // eslint-disable-line @typescript-eslint/no-empty-function
-
+  Object.defineProperty(window, 'location', {
```

Comment:
The line do reset the window location wasn't actually doing anything without this. Returning a Switch instead of just UnconfiguredScreen in ElectionManager made it so that the URL from the previous test was getting incorrectly carried over to the next test without having this reset working. In manual testing I didn't notice any difference in routing.